vcs/Canvas.py

```diff
@@ -349,12 +349,13 @@ class Canvas(object):
         'enableLogo',
         'backend',
         'configurator',
+        '_Canvas__last_plot_actual_args',
+        '_Canvas__last_plot_keyargs',
         '__last_plot_actual_args',
         '__last_plot_keyargs',
         '_continents',
         '_continents_line',
         '_savedcontinentstype',
-        '__weakref__',
```

Comment:
grep-ing through sources seems that we only hold a weak reference to a VTKVCSBackend object. So it seems safe to delete this one.